src/DIRAC/Core/Utilities/DIRACScript.py

```diff
@@ -0,0 +1,126 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+__RCSID__ = "$Id$"
+
+from collections import defaultdict
+import functools
+
+import six
+
+from DIRAC import gLogger
+
+
+class DIRACScript(object):
```

Comment:
This is the class that does the magic for replacing `console_scripts` functions with the highest priority extension. It does nothing when installed with Python 2 (under the assumption that it's a legacy install using PYTHONPATH).